### PR TITLE
feat(RTE): Add Subscript and Superscript Plugins

### PR DESCRIPTION
**Pull request by imoutaharik**

### CODE REVIEW

**Comment by triggertoo** [comment]
Screenshot 2023-05-04 at 11 37 06

Mark can currently be subscript oder superscript ;)

**Comment by triggertoo** [comment]
"clear formatting" should remove superscript and subscript

File: src/components/RichTextEditor/Plugins/SubscriptPlugin/SubscriptButton.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+export const SubscriptButton = ({ editor, id }: PluginButtonProps) => (
+    <ButtonWrapper id={id}>
+        <MarkToolbarButton
+            tooltip={getTooltip('Subscript')}
+            type={getPluginType(editor, id)}
+            icon={<IconStylingWrapper icon={<IconSubscript16 />} />}
+            classNames={getButtonClassNames()}
+            styles={buttonStyles}
+            actionHandler="onMouseDown"
```
[the remark this answers is not in the document]
Comment (imoutaharik) [reply src/components/RichTextEditor/Plugins/SubscriptPlugin/SubscriptButton.tsx:20]: Added!

**Review by triggertoo — APPROVED** [review]
nicely done 👏

**Pull request merged**